src/components/Display.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ value, onChange }) => {
+  const handleChange = e => {
+    if (e.target.value === '') {
+      onChange(0);
+    } else {
+      onChange(parseFloat(e.target.value));
+    }
+  };
+
+  return (
+    <input
+      id="text17"
+      type="number"
+      pattern="[0-9]"
+      autoComplete="off"
+      value={Number(value).toString()}
+      onChange={handleChange}
+    />
+  );
+};
+
+
+Display.propTypes = {
+  value: PropTypes.number.isRequired,
+  onChange: PropTypes.func.isRequired,
```

Comment:
You did a good job adding a few more components.
Note: for every new component or a new way of building the project as to go with the requirement specified by Microverse.

Always read the pathwright instructions before starting the project to avoid the need of sending another code review, and always try as much as possible to stick to the requirements.
Issues:
1. seems you're having `value` as `result`
2. `result` should be a string
3. result default value should be set to 0.

Screenshot from pathwright: 
![image](https://user-images.githubusercontent.com/31897434/83470185-80f8ab80-a479-11ea-99ba-38ce5665a00d.png)